mlx-swift-examples.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -15,7 +15,7 @@
       "location" : "https://github.com/ml-explore/mlx-swift",
       "state" : {
         "branch" : "main",
-        "revision" : "cadf5f8187ac0894e66cd288217e2eda9f2c933d"
+        "revision" : "fbe215ae29ec286bdc66f4b3423e3eea1b3ef2fc"
```

Comment:
Aha! I wonder if this file should be checked in?  I am not sure what the best practice is here, but I agree this is needed!